docs/guides/Package-authoring-best-practices.md

```diff
@@ -0,0 +1,181 @@
+
+Highly inclusive packages are cross-platform and compatible with as many frameworks as possible. Today, that means using .NET Standard. .NET Standard is a specification of .NET APIs that are available on all .NET implementations.
+
+✔️ DO start with including a `netstandard2.0` target.
+> Most general-purpose libraries should not need APIs outside of .NET Standard 2.0. .NET Standard 2.0 is supported by all modern platforms and is the recommended way to support multiple platforms with one target.
+
+See the .NET [Cross-platform targeting guidance](https://docs.microsoft.com/en-us/dotnet/standard/library-guidance/cross-platform-targeting) for more advanced suggestions regarding cross-platform and multitargeting scenarios.
+
+## Package metadata
+
+Metadata is a foundational component of any NuGet package. The quality of your metadata can vastly influence the discoverability and usability of your package.
+
+In Visual Studio, the recommended way to specify package metadata is to go Project > [Project Name] Properties > Package.
+
+Package metadata elements can also be specified directly in the project file (.csproj).
+
+Below is a table mapping and describing available packages metadata elements:
+
+| Visual Studio property name | csproj/MSBuild property name | Nuspec property name        | Description                                                                                                       |
+|-----------------------------|------------------------------|-----------------------------|-------------------------------------------------------------------------------------------------------------------|
+| `Package id`                | `PackageId`                  | `id`                        | The package identifier. A prefix from the identifier can be reserved if it meets the  criteria.                   |
+| `Package Version`           | `Version`                    | `version`                   | NuGet package version. For more information, see NuGet package version.                                           |
+| `Authors`                   | `Authors`                    | `authors`                   | A comma-separated list of package authors, matching the profile names on NuGet.org.                               |
+| `Description`               | `Description`                | `description`               | A long description of the package displayed in UI.                                                                |
+| `Copyright`                 | `Copyright`                  | `copyright`                 | Copyright details for the package. language.                                                                      |
+| `Licensing - Expression`    | `PackageLicenseExpression`   | `license type="expression"` | An SPDX license expression or path to a license file within the package, often shown in UIs like NuGet.org.       |
```

Comment:
Link to documentation: https://docs.microsoft.com/en-us/nuget/reference/nuspec#license

My concern is that a user will not be able to follow those instructions without a lot of bing seaches